app/ts/store/tokens.ts

```diff
@@ -74,7 +74,7 @@ export function useTokenQuery() {
 				const name = await contract.name()
 				const symbol = await contract.symbol()
 				const decimals = await contract.decimals()
-				return { chainId, name, symbol, decimals, address: tokenAddress.value } as const
+				return { chainId, name, symbol, decimals: Number(decimals), address: tokenAddress.value } as const
```

Comment:
`bigint` is the correct type for `decimals` as they must be integer values, and the contract returns an integer.  We should instead fix the problem of handling `bigint`s elsewhere in the stack rather than converting to `number` here.